### PR TITLE
AC Automated Test - LRAC-10151 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10151

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase [hunk 1]
```diff
@@ -331,4 +331,112 @@ definition {
...
+
+		task ("Navigation to event analysis") {
+			ACNavigation.exitSettings();
+
+			ACNavigation.goToEventAnalysis();
+		}
+
+		task ("Add the custom event to analysis") {
+			ACEventAnalysis.addEvent(customEventName = "${customEventName}");
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:431]: hi @ViniLopes20 , what about renaming this task to:

`task ("Check the interactions were not reflected on the event analysis while the custom event was blocked") {`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase [hunk 2]
```diff
@@ -331,4 +331,112 @@ definition {
...
+
+		task ("Check the interactions are not reflected on the event analysis because the custom event is block") {
+			ACTimeFilter.clickTimeFilterButton();
+
+			ACTimeFilter.setLast24Hours();
+
+			ACEventAnalysis.viewAnalysisInformation(informationValueList = "1");
+		}
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:439]: hi @ViniLopes20 , what do you think if we make a new interaction after unblocking this event? So we can make sure the interactions are processed again when validating the `informationValueList =  "2"`, for example...

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsUnblockEvents.testcase [hunk 3]
```diff
@@ -218,7 +218,7 @@ definition {
 			ACCustomEvents.blockCustomEvent(customEventName = "${customEventName}");
 		}
 
-		task ("Navigation to segments") {
+		task ("Navigation to event analysis") {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsUnblockEvents.testcase:221]: Good catch !

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase (hunk 2 above)
Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:439]: Hi @alexcandidos, yes I thought it was a good idea, I was thought if this step could be some other test from our test map, I believe there is no other one of this type, but I will check and change this test

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**